src/isolatable-hybrid.store.ts

```diff
@@ -11,23 +11,23 @@ export type ServingStoreUpdateStrategy = 'always' | 'expired' | 'empty';
  */
 export class IsolatableHybridConfigurationStore<T> extends HybridConfigurationStore<T> {
   constructor(
-    servingStore: ISyncStore<T>,
-    persistentStore: IAsyncStore<T> | null,
+    private readonly _servingStore: ISyncStore<T>,
+    private readonly _persistentStore: IAsyncStore<T> | null,
```

Comment:
Nit: The underscore prefix was big in plain JavaScript, but since TypeScript has proper access modifiers, I haven't seen it used as much anymore. It doesn't look like we're using this convention in a lot of other places either from what I'm seeing.

Reply:
My understanding is that it has been repurposed to signal to typescript you know a variable is unused.

Agree that we should not be using it for `private` properties and methods now.

Reply:
> My understanding is that it has been repurposed to signal to typescript you know a variable is unused.

Oh yeah, that's a thing

Reply:
we're only using undersscores here because the name conflicts with private properties of `HybridConfigurationStore` that have the same name and can't be reused. I could rename this to something else without the underscore or update `HybridConfigurationStore` in the common library to make it `protected` instead. any preferences?

Reply:
ah ok -- I see now. Nope, no preferences. This was really a non-issue to begin with.l

Reply:
Ah yeah in that case I'd make protected as we'd want to use the parent class's two stores anyway I reckon